website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -44,57 +89,235 @@ The [Whole Genome Germline Single Sample workflow](https://github.com/broadinsti
+| ontamination_sites_bed | DNASeqSingleSampleReferences (references) | Contamination site files for the CheckContamination task. | File |
+| contamination_sites_mu | DNASeqSingleSampleReferences (references) | Contamination site files for the CheckContamination task. | File |
+| calling_interval_list | DNASeqSingleSampleReferences (references) | Interval list used for variant calling | File |
+| reference_bin | DragmapReference (dragmap_reference) | Binary representation of the reference FASTA file used for the DRAGEN mode DRAGMAP aligner. | File |
+| hash_table_cfg_bin | DragmapReference (dragmap_reference) | Binary representation of the configuration for the hash table used for the DRAGEN mode DRAGMAP aligner. | File |
+| hash_table_cmp | DragmapReference (dragmap_reference_ | Compressed representation of the hash table that is used for the DRAGEN mode DRAGMAP aligner. | File |
+| haplotype_scatter_count | VariantCallingScatterSettings (scatter_settings) | Scatter count used for variant calling. | Int |
+| break_bands_at_multiples_of | VariantCallingScatterSettings (scatter_settings) | Breaks reference bands up at genomic positions that are multiples of this number; used to reduce GVCF file size. | Int |
+| preemptible_tries |  PapiSettings (papi_settings) | Number of times the workflow can be preempted. | Int | 
+| agg_preemptible_tries |  PapiSettings (papi_settings) | Number of preemtible machine tries for the BamtoCram task. | Int |
+ 
+#### Additional inputs
+Additional inputs that are not contained in a struct are described in the table below. Similar to the struct inputs, these inputs are specified in the [example configuration files](https://github.com/broadinstitute/warp/tree/develop/pipelines/broad/dna_seq/germline/single_sample/wgs/input_files). 
 
-Learn more about the software tools implemented in these tasks by reading the GATK [data pre-processing](https://gatk.broadinstitute.org/hc/en-us/articles/360035535912) and [germline short variant discovery](https://gatk.broadinstitute.org/hc/en-us/articles/360035535932) documentation.
 
+| Input name | Input description | Input type |
+| --- | --- | --- |
+| fingerprint_genotypes_file | Genotype VCF if optionally performing fingerprinting.  | File |
+| fingerprint_genotypes_index | Optional index for the fingerprinting VCF. | File |
+| wgs_coverage_interval_list | Interval list for the CollectWgsMetrics tool. | File |
+| provide_bam_output | If set to true, provides the aligned BAM and index as workflow output; default set to false. | Boolean |
+| use_gatk3_haplotype_caller | Uses the GATK3.5 HaplotypeCalller; default set to true. | Boolean |
+| dragen_functional_equivalence_mode | Boolean used to run the WGS pipeline in a mode functionally equivalent to DRAGEN; set to false by default. | Boolean |
+| dragen_maximum_quality_mode | Boolean used to run the pipeline in DRAGEN mode with modifications to produce maximum quality results; set to false by default. | Boolean |
```

Comment:
What happens if I set `dragen_maximum_quality_mode` to `true` AND `dragen_functional_equivalence_mode` to `true`?